### ISSUE TITLE
Binary files (*.png, *.ico, *.icns) listed as modified upon cloning the repository

### ISSUE THREAD

**Issue by PAN-Chuwen** [ollama/ollama#6274]
#### Steps to Reproduce

1. Clone the repository:

   ```sh
   git clone https://github.com/ollama/ollama.git
   
   cd ollama
   ```

2. Check the status of the repository:
	```sh
   git status
	```
#### Expected Behavior

No files should be listed as modified immediately after cloning the repository.

#### Actual Behavior

The following binary files are listed as modified:

```
modified:  app/assets/app.ico

modified:  examples/modelfile-mario/logo.png

modified:  macapp/assets/icon.icns

modified:  macapp/assets/iconDarkTemplate.png

modified:  macapp/assets/iconDarkTemplate@2x.png

modified:  macapp/assets/iconDarkUpdateTemplate.png

modified:  macapp/assets/iconDarkUpdateTemplate@2x.png

modified:  macapp/assets/iconTemplate.png

modified:  macapp/assets/iconTemplate@2x.png

modified:  macapp/assets/iconUpdateTemplate.png

modified:  macapp/assets/iconUpdateTemplate@2x.png
```

#### Environment

- Tested on macOS and a fresh Ubuntu 24.04 EC2 instance.

#### Analysis

The issue seems to be related to the handling of binary files. It appears that the `.gitattributes` file was not set correctly, causing Git to treat these binary files as text files.

#### Resolution

After adding the following lines to the `.gitattributes` file, the problem was resolved:

```
*.ico binary
*.png binary
*.icns binary
```

You can verify this by cloning the forked repository:

```sh
git clone https://github.com/PAN-Chuwen/ollama.git
```

#### Unclear Aspects

- The exact cause of why Git treated these binary files as text files.
- How Git handles LF/CRLF line endings in binary files.

### OS

Linux, macOS

**Comment by rick-github** [comment]
https://github.com/ollama/ollama/pull/6235

**Comment by FellowTraveler** [comment]
#6183

**Comment by sammcj** [comment]
Yeah I see this happen when people use Windows for development some times. This caused me [headaches](https://github.com/ollama/ollama/pull/5894#issuecomment-2277316957) when trying to update a PR yesterday.

**Comment by PAN-Chuwen** [comment]
Issue resolved in #6235.
